tree/30_generic_methods/service_check_started_at_boot.cf

```diff
@@ -29,16 +29,16 @@ bundle agent service_check_started_at_boot(service_name)
 {
   vars:
 
-    pass2.systemctl_utility_present.!broken_systemctl::
+    pass1.systemctl_utility_present.!broken_systemctl::
```

Comment:
"command_to_check" will be undefined on  pass1.systemctl_utility_present.broken_systemctl